--- a/src/layouts/Blank/index.jsx
+++ b/src/layouts/Blank/index.jsx
@@ -41,3 +41,7 @@
-    const { title = { zh: t('Cloud'), en: 'Cloud' } } = this.info;
-    const { getLocaleShortName } = i18n;
-    const language = getLocaleShortName();
-    return title[language] || t('Cloud') || 'Cloud';
+    const {
+      title = {
+        zh: t('TCS SovereignSecure Cloud'),
+        en: 'TCS SovereignSecure Cloud',
+      },
+    } = this.info;
+    const { getLocaleShortName } = i18n;
+    const language = getLocaleShortName();
+    return (
+      title[language] ||
+      t('TCS SovereignSecure Cloud') ||
+      'TCS SovereignSecure Cloud'
+    );
